modules/nw-allocate-load-balancers-to-subnets-procedure.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// *networking/allocating-load-balancers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-allocating-load-balancers-to-subnets-procedure_{context}"]
+= Specifying AWS subnets for OpenShift API and ingress load balancers at installation
+
+Perform the following steps to allocate API and ingress load balancers to specific subnets.
+
+.Prerequisites
+
+Before you begin, ensure you have:
+
+* An existing AWS virtual private cloud (VPC).
+
+* Pre-configured AWS subnets intended for use by the OpenShift cluster, with the following considerations:
+
+** You have a list of their subnet IDs (for example, `subnet-0123456789abcdef0`). These IDs will be used in the `install-config.yaml` file.
+
+** Use subnets spanning at least two availability zones (AZs) for high availability of load balancers and other critical components, like control planes.
+
+** You have sufficient available IP addresses within these subnets for all assigned roles.
+
+** The AWS configuration for these subnets, including network ACLs and security groups, must permit necessary traffic for all roles assigned to them. For subnets hosting an ingress controller, this typically includes TCP ports 80 and 443 from required sources.
+
+You have the OpenShift installer binary for your target OpenShift version.
+
+* You have an `install-config.yaml` file.
+
+.Procedure
+
+. Prepare the `install-config.yaml` file:
++
+If you haven't already, generate the installation configuration file using the OpenShift installer:
++
+[source,terminal]
+----
+$ openshift-install create install-config --dir=<your_installation_directory>
+----
++
+This command creates the `install-config.yaml` file in the specified directory.
+
+. Define Subnets and assign roles:
```

Comment:
```suggestion
. Define subnets and assign roles:
```
I wouldn't hold up merging for this but if you are fixing the bullet issue, can you also fix this?